pkg/diagnostics/networkpod/external.go

```diff
@@ -0,0 +1,44 @@
+package network
+
+import (
+	"fmt"
+
+	kexec "k8s.io/kubernetes/pkg/util/exec"
+
+	"github.com/openshift/origin/pkg/diagnostics/types"
+)
+
+const (
+	CheckExternalNetworkName = "CheckExternalNetwork"
+)
+
+// CheckExternalNetwork is a Diagnostic to check accessibility of external network within a pod
+type CheckExternalNetwork struct {
+}
+
+// Name is part of the Diagnostic interface and just returns name.
+func (d CheckExternalNetwork) Name() string {
+	return CheckExternalNetworkName
+}
+
+// Description is part of the Diagnostic interface and just returns the diagnostic description.
+func (d CheckExternalNetwork) Description() string {
+	return "Check that external network is accessible within a pod"
+}
+
+// CanRun is part of the Diagnostic interface; it determines if the conditions are right to run this diagnostic.
+func (d CheckExternalNetwork) CanRun() (bool, error) {
+	return true, nil
+}
+
+// Check is part of the Diagnostic interface; it runs the actual diagnostic logic
+func (d CheckExternalNetwork) Check() types.DiagnosticResult {
+	r := types.NewDiagnosticResult(CheckExternalNetworkName)
+
+	externalAddress := "www.redhat.com"
```

Comment:
We may want to use something different here is this is going to be a real, supported test